ldj-readme-renderer/README_template.md

```diff
@@ -0,0 +1,66 @@
+[![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=sqale_rating)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
+# Living Documentation Java
+
+This project is an extension of the [Living Documentation][ldoc] set of tools to
+Java. As part of the Living Documentation toolchain, it can be used to generate
+documentation and diagrams from source code so that documentation is always
+up-to-date.
+
+## State of the project
+
+As of right now, this project is linked to a student assignment originating from the University of
+Twente and InfoSupport. The aim of the project is to make the [LivingDocumentation][ldoc] project
+capable to handle multilingual projects. One aspect of this is to make LivingDocumentation
+available for the Java environment; which is the purpose of this repository.
+
+Within the aspect of making LivingDocumentation available for the Java environment, two features
+can be identified (which are copied from [LivingDocumentation][ldoc] description):
+
+* [**Analyzer**](#analyzer): A tool to analyze Java projects.
+* [**Libraries**](#support-libraries): Assists in generating applications that can create plain text files such as MarkDown, AsciiDoc, PlantUML, Mermaid, and more.
+
+> **Note**: As this repository is currently association to a University assignment, it is unfortunately not possible to directly contribute to this project. 
+> Furthermore, some of the described functionalities may not yet be finalised/are subject to change.
+
+## Getting started
+
+The process of rendering documentation can be summarized as follows:
+1. **Analysis**: Run the Analyzer over the source code of your project. This creates a LivingDocumentation JSON file.
+2. **Render the documentation**: Use a (custom) renderer that reads the generated JSON file and outputs documentation
+
+> Renderers are applications that you can write yourself making use of the support _Libraries_. These interpret the analyzed code and create documentation files.
+
+## Analyzer
+
+The analyzer takes Java source code as input and generates a JSON file (containing a representation 
+of the AbstractSyntaxTree which is compatible with other LivingDocumentation applications).
+
+It can be used either as a Java command line application or as a Maven plugin.
+
+### Command line interface
+The Java analyzer can be invoked directly as a plain Java application [analyzer-java](/analyzer-java). It accepts the following command
+line options:
+
+[~cmd-line-options]
+
+### Maven plugin
+The analyzer can also be used as a Maven plugin (which is defined in [ldj-maven-plugin](/ldj-maven-plugin)).
+For an example on how to use the Maven plugin, see the `pom.xml` configuration in [ldj-maven-plugin-example](/ldj-maven-plugin-example).
+
+### State of analyzer
```

Comment:
Main reason was to address questions like _"What types of classes and relations can this tool model?"_. This gives a dynamic list of the java features that are covered by the analyzer. Relation to the JavaParser documentation is to give a description of the exact java feature (which is not generally deductible from our code I think). In the future this sort of documentation could be replaced by or linked to the JSON schema documentation; answering the above question.

Reply:
Hmm, right - I think I get what you mean, but if this is the intention, the list is not accurate, right? It includes `CatchClause,` but we do not include information about catch clauses in the JSON. I think we actually can not do so, without breaking compatibility; so listing it as something we already do seems odd to me.

If the hyperlinks are currently not useful, but will be useful in the future, I would suggest removing them for now and adding them at that future point.

Reply:
As talked about, right now this list is excluded. If in a later stadium it might be useful to again include this information (albeit in some other documentation artefact), we can revert (or copy from the inverse of ) commit `fff85fa0339fcb6100e516939d0fc3bac867e460` 

Furthermore, this readme now contains a sort of placeholder reference to the JSON schema documentation which is yet to be approved/merged

Reply:
Note for future reference:  commit `ee5deea9df4c85eab988b5098042fb2d5573cc77` now also contains a deletion of a required statement for this feature